### PR TITLE
Retry until PV creation succeeds in `reclaim-storage` playbook

### PR DESCRIPTION
**Pull request by Zempashi**
Fixes: #321

### CODE REVIEW

**Comment by Zempashi** [comment]
@bert-e reset

File: roles/reclaim_local_storage/tasks/main.yml [hunk 1]
```diff
@@ -65,3 +65,6 @@
   loop_control:
      label: "{{ item.metadata.name }}"
   with_items: '{{ released_pv.values()|list }}'
+  register: pv_recreation
+  until: pv_creation is success
```
Comment (NicolasT) [inline roles/reclaim_local_storage/tasks/main.yml]: `pv_creation` vs. `pv_recreation` being registered?

File: ChangeLog.rst [hunk 2]
```diff
@@ -56,6 +56,8 @@ deploy instead of falling back to the chart default (:ghpull:`197`)
 
 :ghissue:`220` - 'Kubernetes Calico (Alternative)' dashboard doesn't work (:ghpull:`221`)
 
+:ghissue:`321` - race condition in CI (:ghpull:`319`)
```
Comment (NicolasT) [inline ChangeLog.rst]: Consider replacing this with the current PR title.

**Review by NicolasT — CHANGES_REQUESTED** [review]

File: roles/reclaim_local_storage/tasks/main.yml (hunk 1 above)
Comment (Zempashi) [reply roles/reclaim_local_storage/tasks/main.yml]: indeed

File: ChangeLog.rst (hunk 2 above)
Comment (Zempashi) [reply ChangeLog.rst]: We don't use the title of the issue for a bug fix ?

**Comment by Zempashi** [comment]
@bert-e reset

**Review by NicolasT — APPROVED** [review]

**Pull request merged**